examples/added_date.py

```diff
@@ -0,0 +1,75 @@
+"""Get machines and the date they were added"""
+
+import os
+from configparser import ConfigParser
+import xlsxwriter  # pylint: disable=import-error
+from cbw_api_toolbox.cbw_api import CBWApi
+    
+CONF = ConfigParser()
+CONF.read(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', 'api.conf'))
+CLIENT = CBWApi(CONF.get('cyberwatch', 'url'), CONF.get('cyberwatch', 'api_key'), CONF.get('cyberwatch', 'secret_key'))
+
+#Parse the date directly from the fiels created_at
+def date_parser(str):
+        t=str.split("-")
+        year=t[0]
+        month=t[1]
+
+        day1=t[2]
+        day=day1[0]+day1[1]
+
+        return day+"/"+month+"/"+year
+                
+#Parse the time of creation directly from the field created_at
+def get_hour_created(str):
+        t=str.split(":")
+        a=len(t[0])
+        b=a-1
+        h=t[0]
+
+        hour=""+h[b-1]+h[a-1]
+        minute=t[1]
+
+        return hour+":"+minute
+
+#Test de la connexion client    
+print("Testing connexion")
+CLIENT.ping()
+
+def server_list_created():
+
+    SERVERS = CLIENT.servers()
+
+    EXPORTED = xlsxwriter.Workbook('export.xlsx')
+
+    # Create each category tab
+    COMPUTER = EXPORTED.add_worksheet("Computers")
+
+    # Build a list with each server and it's details
+    SERVERS_LIST = []
+    for server in SERVERS:
+        SERVERS_LIST.append(server)
```

Comment:
Why you need a servers_list ? Client.servers() is not a list ?

Reply:
It's a list. And it works by listing directly Client.Servers() instead of creating a second list. It is just a habits